File: programme/check-prime-number/check-prime-number.go
// Write a programmeto check prime number
package main

import "fmt"

func main() {
	var n int
	fmt.Print("Enter the number : ")
	fmt.Scan(&n)
	fmt.Println("\nThe number is Prime :", isPrime(n))
}

func isPrime(n int) bool {
	for i := 2; i <= n/2; i++ {
		if n%i == 0 {
			return false
		}
	}
	return true
}

File: programme/check-prime-number/check-prime-number.jl
# Write a programme to check prime number

print("Input  : ")
number = parse(Int32, readline())

function checkPrime(num)
    flag = false
    if num == 0 || num == 1
        print("Output : Not prime number\n")
    else
        for i = 2:trunc(Int, num // 2)
            if num % i == 0
                flag = true
                print("Output : Not prime number\n")
            end
        end
        flag == false && print("Output : Prime number\n")
    end
end

checkPrime(number)
